Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "sd-jwt-rs"
+name = "sd_jwt_rs"
```

Comment:
In Java this import is not convenient.
`import sd_jwt_rs.*`

Let`s pick another name without camel case or underscores